react/components/FilterBar/index.js

```diff
@@ -217,6 +220,10 @@ export const filterBarPropTypes = {
   submitFilterLabel: PropTypes.string,
   /** New Filter title label for inside the 'More options' menu */
   newFilterLabel: PropTypes.string,
+  testIds: PropTypes.shape({
+    moreOptionsButton: PropTypes.string,
+    submitFiltersButton: PropTypes.string,
+  }),
```

Comment:
These should be inside a `moreOptionsFilterTag` object, no? This way if we wanted to add ids to clearAllFilterButton we could only add a new prop to testIds